EDITORIAL_GUIDELINES.md

```diff
@@ -0,0 +1,81 @@
+## Editorial Style Suggestions
+This document provides editorial style suggestions for the FOCUS v1.0 specifications. It was created in response to a recent discussion about the need for a consistent and professional editorial style to enhance and harmonize FOCUS technical specifications. The guidelines outlined below aim to standardize the presentation of various elements such as **column names**, *column IDs*, [_glossary terms_](), **NORMATIVE KEYWORDS**, **`attribute names`**, *`attribute IDs`* and other key components of the FOCUS documentation.
+
+1. **Column Names**
+   - **Style**: **Bold**, (_with blank space for compound names_)
+   - **Reasoning**: Column names should be easily distinguishable for quick reference and clarity.
+   - **Example**: **Billing Account ID**, **Resource Name**
+
+2. **Column IDs**
+   - **Style**: *Italic*, (_without blank space for compound names, and compound words capatilized_)
+   - **Reasoning**: Italicizing the column IDs helps differentiate them from the display names and other text, making it clear when referring to specific IDs.
+   - **Example**: *BillingAccountId*, *ResourceName*
+
+3. **Glossary Terms**
+   - **Style**: [_Underlined_](), and capitalized
+   - **Reasoning**: Underlining glossary terms makes them stand out, helping readers quickly locate definitions and understand terminology.
+   - **Example**: [_Amortization_](), [_CommitmentDiscountId_]()
+
+4. **Normative Statements (e.g., MUST, SHOULD, MAY)**
+   - **Style**: **ALL CAPITALS** and **bold**
+   - **Reasoning**: Using all capitals for normative terms emphasizes their importance and aligns with standard practice in technical specifications to indicate requirement levels. In alignment with [BCP14](https://www.rfc-editor.org/info/bcp14) [[RFC2119](https://datatracker.ietf.org/doc/html/rfc2119)][[RFC8174](https://datatracker.ietf.org/doc/html/rfc8174)].
+   - **Example**: **MUST**, **SHOULD**, **MAY**
+
+5. **Descriptions and Content Constraints**
+   - **Style**: Normal text, but ensure clear headings and subheadings
+   - **Reasoning**: These sections should remain easily readable, with clear headings for each sub-section to facilitate navigation.
+   - **Example**:
+     - **Description**: A name assigned to a grouping of resources or services, often used to manage access and/or cost.
+     - **Content Constraints**:
+       - **Column type**: Dimension
+       - **Feature level**: Conditional
+
+6. **Attribute Names and IDs in Specifications**
```

Comment:
Attributes are specified in a similar pattern to column definitions.. why does name and id matter but not the other sections?